src/components/Editor/EditPublications.vue

```diff
@@ -62,7 +58,8 @@
                 <div> <b>PUBLISHED IN:</b> {{ publication.journal }}, {{ publication.year }}</div>
                 <div> <b>AUTHORS:</b> {{ publication.authors }}</div>
                 <v-switch
-                  v-model="publication.isCitation"
+                  :input-value="publication.isCitation"
+                  @change="toggleCitation(pubIndex)"
```

Comment:
here eslint suggest to order directives this way:

                  :input-value="publication.isCitation"
                  color="green"
                  label="Cite record using this publication?"
                  @change="toggleCitation(pubIndex)"

![image](https://user-images.githubusercontent.com/25683981/105479873-e7e30300-5c9c-11eb-9129-494ac9611ac8.png)